syntax = "proto3";

package msg.net;

message AddSubscriber {
  uint32 id_message = 1;
}

message RemoveSubscriber {
  uint32 id_message = 1;
}
